.gitignore

```diff
@@ -1 +1,4 @@
 target/
+.metals
+project/.bloop/
+.bloop/
```

Comment:
If these are the folders that your IDE creates I don't think this is a good place for them.

General rule I use for `.gitignore` is that all things specific to developer's environment should not go into the project's `.gitignore` but into your global `.gitignore` instead. Otherwise we will end up with a crazy mix of ignores because everyone uses different tools, and will lose track of where certain things came from.

Example:
```
$ git config -l --global
core.excludesfile=/home/smikhtoniuk/.gitignore
...
```
```
cat /home/smikhtoniuk/.gitignore
.idea/
```

Reply:
Got it, thanks.